unit1/lesson13.rst

```diff
@@ -0,0 +1,36 @@
+Unit 1 - Lesson 2 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Add styles to static HTML pages using CSS.
+- Work with Git to version control personal work.
+- Learn how Git can be used to manage teamwork.
+
+Objectives
+----------
+
+CSS
+^^^
+
+- Explain how CSS allows for separation of display and document structure
+- Describe the 3 locations that style rules may be placed: external, document, inline
+- Use CSS rules at each of the 3 possible locations
+- Describe precedence between CSS rules based on location
+- Use basic CSS rule types: font-size, color, font-family, background-color, text-align, etc
+- Describe the following simple selector types, and situations in which each is preferred: element, ID, class
+
+Git
+^^^
+
```

Comment:
From my own experience going through LC101 and Liftoff, I think it would be beneficial to provide a more detailed explanation about the differences between 'git fetch' and 'git pull'.  When is it appropriate to use one over the other?

Reply:
I think that is more of a Liftoff topic. The best way to understand the difference between git fetch and git pull is in the context of a project. Students won't use Git very collaboratively until Liftoff.